user-docs/documentation/docusaurus.config.js

```diff
@@ -74,6 +80,26 @@ const config = {
         searchPagePath: 'search',
       },
     }),
+  plugins: [
+    [
+      'docusaurus-plugin-typedoc-api',
+      {
+        projectRoot: path.join(__dirname, '../..'),
+        packages: [
+          {
+            path: 'packages/serverless-contracts',
+            entry: {
+              index: 'src/index.ts',
+            },
+          },
+        ],
+        minimal: true,
+        readmes: true,
+        debug: true,
+        tsconfigName: 'tsconfig.docs.json',
```

Comment:
I think it would be better to place this config in the `serverless-contracts` package, what do you think?